src/main/java/com/nha/abdm/wrapper/hiu/hrp/consent/requests/callback/ConsentStatus.java

```diff
@@ -12,7 +13,13 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class ConsentStatus {
-  private String id;
+  /**
+   * Added JsonIgnore because while fetching the consentStatus there are two id
+   * 1) consentRequestId
+   * 2) consentId in consentArtifacts both are represented as ID
+   * so the consentRequestId is not required for user, so ignoring the consentRequestId
+   */
+  @JsonIgnore private String id;
```

Comment:
nit: lets have private String id in a separate line

Reply:
The spotless modifies the spring annotations into a single line but modified the changes.